packages/graphql-language-service/package.json

```diff
@@ -31,13 +31,22 @@
     "graphql": "^15.5.0 || ^16.0.0"
   },
   "dependencies": {
-    "graphql-language-service-interface": "^2.10.2",
-    "graphql-language-service-parser": "^1.10.4",
-    "graphql-language-service-types": "^1.8.7",
-    "graphql-language-service-utils": "^2.7.1"
+    "graphql-config": "^4.1.0",
+    "vscode-languageserver-types": "^3.15.1",
+    "nullthrows": "^1.0.0",
+    "@types/json-schema": "7.0.9"
```

Comment:
This dep should probably be in devDeps

Reply:
It’s used for user facing interfaces, so users have complained about this in the past with other types. What do you think?

Reply:
I personally think its not a great call

Reply:
Or I could just leave this and others as `unknown` and link to the related API doc. I can't imagine it's that useful to have the JSONSchema definition result?

Reply:
If the type is exported, wouldn't that automatically make the type available?

Reply:
the type comes from a third party library's type system, which means that that third party's `@types` library needs to be included in `dependencies` for the type to be available

Reply:
made the reccomended change and updated the docs, should be all good now!